erts/doc/src/erlang.xml

```diff
@@ -11779,6 +11792,13 @@ improper_end</pre>
         <p>Allowed in guard tests.</p>
         <p>Failure: <c>badarg</c> if <c><anno>List</anno></c>
           is the empty list <c>[]</c>.</p>
+        <pre>
+> <input>tl([]).</input>
+** exception error: bad argument
+     in function  tl/1
+        called as tl([])
+        *** argument 1: not a nonempty list
+</pre>
```

Comment:
I don't see the point of this example. I suggest you remove it.

Reply:
done